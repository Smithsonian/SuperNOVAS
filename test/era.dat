
2450545.000000 TEST       S2 O0 A1:    14.848330

2450545.000000 TEST       S2 O1 A1:    14.848330

2450545.000000 TEST       S2 O2 A1:    14.848330

2450545.000000 SUN        S0 O0 A1:    14.848330

2450545.000000 SUN        S0 O1 A1:    14.848330

2450545.000000 SUN        S0 O2 A1:    14.848330

2451545.000000 TEST       S2 O0 A1:   280.460618

2451545.000000 TEST       S2 O1 A1:   280.460618

2451545.000000 TEST       S2 O2 A1:   280.460618

2451545.000000 SUN        S0 O0 A1:   280.460618

2451545.000000 SUN        S0 O1 A1:   280.460618

2451545.000000 SUN        S0 O2 A1:   280.460618

2452545.000000 TEST       S2 O0 A1:   186.072906

2452545.000000 TEST       S2 O1 A1:   186.072906

2452545.000000 TEST       S2 O2 A1:   186.072906

2452545.000000 SUN        S0 O0 A1:   186.072906

2452545.000000 SUN        S0 O1 A1:   186.072906

2452545.000000 SUN        S0 O2 A1:   186.072906
