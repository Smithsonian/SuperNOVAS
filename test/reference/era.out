  -10000.000 A0:   144.337737
       0.000 A0:   280.460618
   10000.000 A0:    56.583499
   10000.000 A0:    56.583499
   10000.010 A0:    60.193355
  -10000.000 A1:   144.337737
       0.000 A1:   280.460618
   10000.000 A1:    56.583499
   10000.000 A1:    56.583499
   10000.010 A1:    60.193355
